int64 motor_speed1
int64 motor_speed2
